src/index.tsx

```diff
@@ -72,6 +73,7 @@ const startApp = async () => {
   render(
     <Router history={history}>
       <ApolloProvider client={apolloClient}>
+        <NostoProvider account="mwp2zpea">
```

Comment:
Can this account be put in some config file or something to easily switch to different merchants?